### PR TITLE
x11-wm/notion version bump

### PR DESCRIPTION
**Pull request by moaxcp**
version bump for the notion window manager. The 9999 ebuild has been updated as well. Development has switched to github from sourceforge.

### CODE REVIEW

**Comment by mgorny** [comment]
Are you the proxy-maintainer for this package?

**Comment by moaxcp** [comment]
No but I can be. I looked into the proxy-maintainer project but this package has not been placed in the orphan packages list.

**Comment by idella** [comment]
It makes us indecisive when the package has a proxy maintainer listed then someone else entirely presents an ebuild, surplanting the intended role of the proxy maintainer who is quite silent.

in metadata.xml
        maintainer
              email little_hobbit@lavabit.com</email>
        /maintainer

**Comment by dset0x** [comment]
@idella FWIW, lavabit.com was shut down years ago, so it's going to be impossible to contact them with that e-mail.

**Comment by idella** [comment]
In my view that favours making @moaxcp a new user or proxy maintianer. He needs contact me via the proxy-maintainers project. We've not attempted contact to the listed maintainer but this type of non response sees them replaced with someone willing.

**Comment by mgorny** [comment]
Indeed. little_hobbit is still using the lavabit account for bugzilla, and has no activity since 2012.

File: x11-wm/notion/notion-9999.ebuild [hunk 1]
```diff
@@ -30,16 +30,16 @@ DEPEND="${RDEPEND}
...
-		-e "/^LDFLAGS=/{s:=:+=: ; s:-Wl,--as-needed::}" \
-		-e "/^CC=/s:=:?=:" \
-		-e "/^PREFIX/s:=.*$:= \${ROOT}usr:" \
-		-e "/^ETCDIR/s:=.*$:= \${ROOT}etc/notion:" \
-		-e "/^LIBDIR/s:=.*:= \$(PREFIX)/$(get_libdir):" \
-		-e "/^DOCDIR/s:=.*:= \$(PREFIX)/share/doc/${PF}:" \
-		-e "/^LUA_DIR/s:=.*$:= \$(PREFIX)/usr:" \
-		-e "/^VARDIR/s:=.*$:= \${ROOT}var/cache/${PN}:" \
-		-e "s:^\(X11_PREFIX=\).*:\1\$(PREFIX)/usr:" \
+                -e "/^CFLAGS/{s:-Os:: ; s:-g::}" \
```
Comment (mgorny) [inline x11-wm/notion/notion-9999.ebuild]: Please set your editor to always use tabs to indent ebuilds. I'm going to fix it this time when committing.

Comment (mgorny) [inline x11-wm/notion/notion-9999.ebuild]: Or not. I see more issues so you will have an opportunity to fix them yourself :).

File: x11-wm/notion/notion-3_p2015061300.ebuild [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/x11-wm/notion/notion-3_p2013030200.ebuild,v 1.4 2014/05/25 13:01:33 jer Exp $
+
+EAPI=5
+
+inherit eutils multilib toolchain-funcs
+
+DESCRIPTION="Notion is a tiling, tabbed window manager for the X window system"
+HOMEPAGE="http://notion.sourceforge.net"
+SRC_URI="https://github.com/raboof/${PN}/archive/${PV/_p/-}.tar.gz"
```
Comment (mgorny) [inline x11-wm/notion/notion-3_p2015061300.ebuild:11]:
```
Saving to: ‘/srv/nfs/common/distfiles/3-2015061300.tar.gz’
```

This is not a very good name for a distfile. Use `->` syntax to rename it to something more unique.

File: x11-wm/notion/notion-3_p2015061300.ebuild [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="nls xinerama +xrandr"
+
+RDEPEND=">=dev-lang/lua-5.1
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXext
+	nls? ( sys-devel/gettext )
```
Comment (mgorny) [inline x11-wm/notion/notion-3_p2015061300.ebuild:22]: Are you absolutely sure it requires gettext at runtime?

File: x11-wm/notion/notion-3_p2015061300.ebuild [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+	xinerama? ( x11-libs/libXinerama )
+	xrandr? ( x11-libs/libXrandr )"
+
+DEPEND="${RDEPEND}
+		virtual/pkgconfig"
```
Comment (mgorny) [inline x11-wm/notion/notion-3_p2015061300.ebuild]: Indent that consistently to RDEP.

**Comment by mgorny** [comment]
Also please switch the live ebuild to `git-r3` eclass, and replace little_hobbit with yourself in metadata.xml.

**Comment by idella** [comment]
I have set you as new proxy maintainer for the package x11-wm/notion.
I also suggest use of dev-lang/lua:0 in RDEPEND. Currently we get:

  dependency.missingslot        2
   x11-wm/notion/notion-3_p2013030200.ebuild: RDEPEND: 'dev-lang/lua' matches more than one slot, please specify an explicit slot and/or use the := or :\* slot operator
   x11-wm/notion/notion-9999.ebuild: RDEPEND: 'dev-lang/lua' matches more than one slot, please specify an explicit slot and/or use the := or :\* slot operator

which similarly applies to the notion-3_p2015061300.ebuild. Features like the inconsistent indent you have clearly inherited form the notion-3_p2013030200.ebuild. They still need re-setting.
You have those changes to set and submit here and / or attach in the bug and I can commit accordingly.

**Comment by moaxcp** [comment]
Ok thanks for all the help. I will work on it this week.

File: x11-wm/notion/notion-9999.ebuild (hunk 1 above)
Comment (moaxcp) [inline x11-wm/notion/notion-9999.ebuild]: Hello. I'm happy to fix things. Is there a style guide for ebuilds somewhere?

File: x11-wm/notion/notion-3_p2015061300.ebuild (hunk 3 above)
Comment (moaxcp) [inline x11-wm/notion/notion-3_p2015061300.ebuild:22]: according to the readme.md it requires gettext. https://github.com/raboof/notion

**Comment by xmw** [comment]
I'm adding this to the tree, closing this here.

**Comment by xmw** [comment]
```
+*notion-3_p2015061300 (05 Aug 2015)
+
+  05 Aug 2015; Michael Weber <xmw@gentoo.org> +notion-3_p2015061300.ebuild,
+  notion-3_p2013030200.ebuild, notion-9999.ebuild:
+  Version bump (bug 523566, p/r 160, thanks  John Mercier, Bailey Kong and
+  idella4).
+
```

**Pull request closed without merging**